src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -987,10 +973,9 @@ private AttributeSyntax ParseAttribute()
                 return (AttributeSyntax)this.EatNode();
             }
 
-            var name = this.ParseQualifiedName();
-
-            var argList = this.ParseAttributeArgumentList();
-            return _syntaxFactory.Attribute(name, argList);
+            return _syntaxFactory.Attribute(
+                this.ParseQualifiedName(),
+                this.ParseAttributeArgumentList());
```

Comment:
note: while there are many fixups like this in teh code, looking through, the majority of the existing constructs already follow the new form, so this is just brining things in line with a proven approach that has the benefits listed in teh prior comments.